packages/cli/commands/logs.js

```diff
@@ -142,16 +62,24 @@ exports.handler = async options => {
       process.exit();
     }
   );
+  const functionId = functionResp.id;
 
   logger.debug(`Retrieving logs for functionId: ${functionResp.id}`);
 
+  let logsResp;
+
   if (follow) {
+    const spinner = ora(
+      `Waiting for log entries for '${functionPath}' on account '${accountId}'.\n`
+    );
+    const tailCall = after => getFunctionLogs(accountId, functionId, { after });
+    const fetchLatest = () => getLatestFunctionLog(accountId, functionId);
     await tailLogs({
-      functionId: functionResp.id,
-      functionPath,
       accountId,
-      accountName: options.portal,
```

Comment:
Do we not need this?

Reply:
If we do want to show the account name instead of the id, we do need to incorporate something like this in the new code. That said, I think the way to handle that is to get the account config using the id so that it works consistently even in cases where the `defaultPortal` is used. We also should be consistent across all commands in terms of how we refer to an account in the output.